doc/nrf/ug_matter_device_certification.rst

```diff
@@ -57,8 +123,74 @@ The attestation must be filled by the person responsible for end product certifi
 
 Once both these requirements are met, the responsible person can download the `Matter Attestation of Security template`_, fill it in, and submit it in the certification web tool when applying for the certification.
 
-Matter certification inheritance
-********************************
+Processing
+==========
+
+During this stage of the certification process, CSA verifies the submission for relevant criteria, including relevant documentation, membership in CSA, completion of testing, and payment of applicable fees.
+
+Certification
+=============
+
+At the end of a successful processing, the Matter component is granted the certification.
+
+Matter certification process variants
+*************************************
+
+
+
+Rapid recertification program
```

Comment:
Add note that this program is experimental and it would be removed in next years.
more info about this program.
https://groups.csa-iot.org/wg/members-all/document/27427